sidebars.js

```diff
@@ -178,21 +187,34 @@ module.exports = {
 		},
 		{
 			type: 'category',
-			label: 'Standards',
+			label: 'Plugin development',
```

Comment:
This should be Development if we want to be consistent with the permalink / slug.